crs_motion_planning/src/motion_planning_server.cpp

```diff
@@ -264,28 +180,33 @@ class MotionPlanningServer : public rclcpp::Node
   void planProcess(std::shared_ptr<crs_msgs::srv::PlanProcessMotions::Request> request,
                    std::shared_ptr<crs_msgs::srv::PlanProcessMotions::Response> response)
   {
-    crs_motion_planning::pathPlanningConfig motion_planner_config = *motion_planner_config_;
+    namespace fs = boost::filesystem;
+    crs_motion_planning::pathPlanningConfig::Ptr motion_planner_config;
+    std::string config_rel_path = "config/motion_planning/MP_config.yaml";
```

Comment:
Can this file path not be passed as a parameter?

Reply:
Yes, that can be adjusted to be a parameter